aiomonitor/monitor.py

```diff
@@ -243,7 +243,7 @@ def map_args(self, cmd: Callable, args: Sequence[str]
         for param in params:
             if (param.annotation is param.empty or
                     not callable(param.annotation)):
-                type_: Callable[[Any], Any] = lambda x: x  # noqa
+                type_ = Callable[[Any], Any] = lambda x: x  # noqa
```

Comment:
It's not a typo, it's type annotation. With your patch this code becomes broken. What are you trying to achieve?